src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -810,6 +810,7 @@ module Library
       # @example
       #   require "library/v1/library_service_api"
       #
+      #   Alignment = Google::Example::Library::V1::SomeMessage2::SomeMessage3::Alignment
```

Comment:
Right, I don't think future changes to improve samples would be considered breaking